input_tool.py

```diff
@@ -0,0 +1,203 @@
+import pickle
+import json
+import os
+import time
+import psycopg2
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# can be modified to the file path of experiment data
+EXPERIMENT_DATA_PATH = os.path.join(os.getcwd(),'local')
```

Comment:
change `'local'` to `'eva'` to reflect change to data directory

Reply:
I would also think about passing the path to `eva/` into the input tool since it won't always be in the current working directory

Reply:
I have the experiment path point to the `local` directory temporarily until all the pkl filenames in `eva` have changed to reflect the new naming convention. Do you know where I can download the new directory with the filename changes?

Reply:
You can still use the experiment path pointing to the `local` directory, just change `local` to `eva` since the files in your `local` directory should be updated with the new naming convention

Reply:
Got it, thanks!